src/Api/Controllers/SecretsController.cs

```diff
@@ -33,7 +33,13 @@ public async Task<ListResponseModel<SecretIdentifierResponseModel>> GetSecretsBy
         return new ListResponseModel<SecretIdentifierResponseModel>(responses);
     }
 
-
+    [HttpGet("projects/{projectId}/secrets")]
```

Comment:
Can't decide if this belongs to secrets controller or projects. Probably projects but it's annoying to drag in secrets concerns in projects.